packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec

```diff
@@ -0,0 +1,40 @@
+%global repo_orgname project-receptor
+%global repo_name satellite-receptor-installer
+
+%global role_orgname %{repo_orgname}
+%global role_name satellite_receptor_installer
+
+Name: ansiblerole-satellite-receptor-installer
+Summary: Packaging of the satellite_receptor_installer Ansible role
+Version: 0.5.0
+Release: 1%{?dist}
+License: GPLv3
+
+Source0: https://github.com/%{repo_orgname}/%{repo_name}/archive/v%{version}.tar.gz#/%{role_name}-%{version}.tar.gz
```

Comment:
I'd *prefer* if the source would be the tarball on galaxy, but the role does not seem to be uploaded to galaxy yet. (not a blocker)

Reply:
it is [here](https://galaxy.ansible.com/project-receptor/satellite_receptor_installer), I just followed what I've seen in other ansible role specs, happy to change it

Reply:
FTR, galaxy does not host roles, just collections, so my comment is moot.